Plots/MapProjections/NCL_maponly_2.py

```diff
@@ -0,0 +1,121 @@
+    - Original NCL plots: https://www.ncl.ucar.edu/Applications/Images/maponly_2_1_lg.png https://www.ncl.ucar.edu/Applications/Images/maponly_2_2_lg.png https://www.ncl.ucar.edu/Applications/Images/maponly_2_3_lg.png
+"""
+
+###############################################################################
+# Import packages
+
+import numpy as np
+import cartopy.crs as ccrs
+import cartopy.feature as cfeature
+import matplotlib.pyplot as plt
+
+from geocat.viz import util as gvutil
+
+###############################################################################
+# Plot:
+
+
+def map_plot(scale, long_min, long_max, lat_min, lat_max, long_labels,
+             lat_labels):
+    """Plots a map-only figure with continent borders, country borders, and
+    lakes at a certain longitude and latitude.
+
+    Parameters
+    ----------
+    scale : :obj:'str':
+        scale of continent borders and lakes
+
+    long_min : :obj:'int':
+        minimum longitude for plotting
+
+    long_max : :obj:'int':
+        maximum longitude for plotting
+
+    lat_min : :obj:'int':
+        minimum latitude for plotting
+
+    lat_max : :obj:'int':
+        maximum latitude for plotting
+
+    long_labels : :obj:'list':
+        list of tick labels for x axis
+
+    lat_labels : :obj:'list':
+        list of tick labels for y axis
+
+    Description
+    -----------
+        Plots a figure with longitude with range long_min to long_max and latitude with
+        range from lat_min to lat_max. The scale of the continent borders and
+        lakes depends on the scale variable. The latitude labels are set by lat_labels.
+    """
+
+    # Define the projection and generate axes using Cartopy
+    projection = ccrs.PlateCarree()
+    # Generate figure with (width, height) in inches
+    fig = plt.figure(figsize=(7, 7))
+    ax = plt.axes(projection=projection)
+
+    # Add in country borders, continent borders, and lakes
+    ax.add_feature(
+        cfeature.NaturalEarthFeature(category='cultural',
+                                     name="admin_0_countries",
+                                     scale="110m",
+                                     facecolor="none",
+                                     edgecolor="black",
+                                     linewidth=0.2))
+    ax.add_feature(cfeature.LAND.with_scale(scale),
+                   edgecolor="#4141a0",
+                   facecolor="none")
+    ax.add_feature(cfeature.LAKES.with_scale(scale),
+                   edgecolor="#4141a0",
+                   facecolor="none")
+
+    # Set extent to show particular area of the map
+    ax.set_extent((long_min, long_max, lat_min, lat_max), crs=projection)
+
+    # Use geocat.viz.util convenience function to add minor and major tick lines
+    gvutil.add_major_minor_ticks(ax,
+                                 x_minor_per_major=3,
+                                 y_minor_per_major=3,
+                                 labelsize=10)
```

Comment:
```suggestion
                                 labelsize=15)
```
Increasing the label size to be closer to NCL but also to make the projections a bit easier to read. After this change I should be ready to approve this PR!